core/templates/dev/head/pages/admin/activities_tab/admin_dev_mode_activities_tab_directive.html

```diff
@@ -0,0 +1,46 @@
+<script type="text/ng-template" id="pages/admin/dev_mode_activities_tab_directive">
+  <md-card class="oppia-page-card oppia-long-text" style="max-width: 700px">
+    <div class="container" style="margin-left: 0;">
+      <h3>Reload a single exploration</h3>
+      {% for exploration in demo_explorations %}
```

Comment:
I don't think it's a good idea to have jinja stuff all over the place in random directives. Can we get rid of it?